src/components/ApplicationForm/About/About.module.scss

```diff
@@ -4,13 +4,16 @@
 @use '../../../styles/vars.module';
 
 .field{
+    label:has(+ input:disabled){
```

Comment:
This isn't entirely supported by all browsers, namely Firefox. Nonetheless, this is probably the best and only way to do such as task.